vector/src/main/java/im/vector/app/core/pushers/PushersManager.kt

```diff
@@ -64,14 +65,16 @@ class PushersManager @Inject constructor(
             gateway: String
     ) = HttpPusher(
             pushKey,
-            stringProvider.getString(R.string.pusher_app_id),
+            appId = stringProvider.getString(R.string.pusher_app_id),
             profileTag = DEFAULT_PUSHER_FILE_TAG + "_" + abs(activeSessionHolder.getActiveSession().myUserId.hashCode()),
-            localeProvider.current().language,
-            appNameProvider.getAppName(),
-            activeSessionHolder.getActiveSession().sessionParams.deviceId ?: "MOBILE",
-            gateway,
+            lang = localeProvider.current().language,
+            appDisplayName = appNameProvider.getAppName(),
+            deviceDisplayName = AppBuildConfig.getModel(),
```

Comment:
You can find the `displayName` of a given device in the `DeviceInfo` model. I created a `useCase` to retrieve all device info related to a given `deviceId`: `GetDeviceFullInfoUseCase`. But since `PushersManager` is in the core module, I don't think this is a good idea to import this use case here, since it is in a feature package. I guess we don't want import from feature into core.
Maybe you can have a new `useCase` in the core module `GetDeviceInfoUseCase` which will rely on: `session.cryptoService().getMyDevicesInfoLive(deviceId).asFlow()`?

And `AppBuildConfig` can be removed then.

Reply:
Implemented this but instead of using `getMyDevicesInfoLive` I opted to use `getMyDevice` because we need specifically only the device being used, not the other devices registered to the account